src/Reveal.Sdk.Dom.Tests/Visualizations/SingleValueLabelsVisualizationBaseFixture.cs

```diff
@@ -0,0 +1,121 @@
+
+            // Assert
+            Assert.Equal(title, visualization.Title);
+            Assert.Equal(dataSourceItem, visualization.DataDefinition.DataSourceItem);
+            Assert.IsType<SingleValueLabelsVisualizationDataSpec>(visualization.VisualizationDataSpec);
+        }
+
+        [Fact]
+        public void GetLabels_ReturnsVisualizationSpecRows_IfDataSpecIsSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new SingleValueLabelsVisualizationDataSpec()
+            {
+                Rows = new List<DimensionColumn>
+                {
+                    new(),
+                    new(),
+                }
+            };
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var labels = visualization.Labels;
+
+            // Assert
+            Assert.Equal(visSpec.Rows, labels);
+        }
+
+        [Fact]
+        public void GetLabels_ReturnsEmptyList_IfDataSpecIsNotSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new CategoryVisualizationDataSpec();
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var labels = visualization.Labels;
+
+            // Assert
+            Assert.Empty(labels);
+        }
+
+        [Fact]
+        public void GetValues_ReturnsVisualizationSpecValue_IfDataSpecIsSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new SingleValueLabelsVisualizationDataSpec()
+            {
+                Value = new List<MeasureColumn>
+                {
+                    new(),
+                    new(),
+                }
+            };
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var values = visualization.Values;
+
+            // Assert
+            Assert.Equal(visSpec.Value, values);
+        }
+
+        [Fact]
+        public void GetValues_ReturnsEmptyList_IfDataSpecIsNotSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new CategoryVisualizationDataSpec();
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var values = visualization.Values;
+
+            // Assert
+            Assert.Empty(values);
```

Comment:
Similar to the Labels test, add Assert.NotNull(values) to make sure that returned list is not null